src/main/webapp/tests.html

```diff
@@ -7,16 +7,21 @@
   <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.10.0.css">
 </head>
 <body>
+    <select id="region-select"></select>
   <div id="qunit"></div>
-  <div id="qunit-fixture"></div>
+  <div id="qunit-fixture">
+
+  </div>
```

Comment:
Is this change intentional?

Reply:
going to put the select dropdown menu here once it's working (it's not here right now because this section hides the element and I want to see the box when debugging)